src/rgw/rgw_d3n_datacache.cc

```diff
@@ -161,9 +161,8 @@ void D3nDataCache::d3n_libaio_write_completion_cb(D3nCacheAioWriteRequest* c)
   { // update cache_map entries for new chunk in cache
     const std::lock_guard l(d3n_cache_lock);
     auto it = d3n_outstanding_write_list.find(c->oid);
-    if (it != d3n_outstanding_write_list.end()) {
+    if (it != d3n_outstanding_write_list.end())
       d3n_outstanding_write_list.erase(it);
-    }
```

Comment:
please keep the {}s around single-line blocks, here and below (see [CodingStyle](https://github.com/ceph/ceph/blob/master/CodingStyle#L92-L103))